mailpoet/tests/acceptance/Forms/EditorCreateCustomFieldCest.php

```diff
@@ -74,32 +92,44 @@ public function createCustomTextInput(\AcceptanceTester $i) {
     $this->checkCustomTextInputInForm($i);
 
     $i->wantTo('Change text input validation');
-    $i->selectOption('[data-automation-id="settings_custom_text_input_validation_type"]', 'Alphanumerical');
+    $i->selectOption('[data-automation-id="settings_custom_text_input_validation_type"]', 'Numbers only');
 
     $i->wantTo('Update label and save the form');
-    $i->fillField('[data-automation-id="settings_custom_text_label_input"]', 'My updated custom text input');
+    $i->fillField('[data-automation-id="settings_custom_text_label_input"]', $customFieldName . ' updated');
```

Comment:
This is just a detail and feel free to ignore it, but maybe since you already refactored this test to use `$customFieldName` instead of hard coding `My custom select` in several places, I wonder if we should do the same here and create a variable like `$updatedCustomFieldName` and used it instead of hard coding `$customFieldName . ' updated'` in several places. What do you think?

Reply:
Thanks for quick review! I've been thinking about this and decided to make it this way just because there is also additional check on the front-end with * sign, in that case it would be `$updatedCustomFieldName . ' *'` so anyway I would need to use that . plus thing.